stats/Record.md

```diff
@@ -44,6 +44,8 @@ measure name instead of the `Measure`.
 
 Implementations MAY define a `MeasurementMap` which describes a set of data points to be collected
 for a set of Measures. Adding this functionality may improve the efficiency of the record usage API.
+Besides, when recording Measurements, `MeasurementMap` should optionally take a map of string-object
+pairs as the attachment of exemplar.
```

Comment:
This is more generic than originally discussed (i.e. Map<String, String>) but I think makes sense. Let's discuss it in the cross-language sync in case non-gc'd languages may have concerns about object lifetimes.

Reply:
Per our discussion today I've reverted the change to use Map<String, String> instead.